src/relayFeeCalculator/chain-queries/factory.ts

```diff
@@ -37,8 +37,10 @@ export class QueryBase__factory {
         symbolMapping,
         spokePoolAddress,
         simulatedRelayerAddress,
+        logger,
         coingeckoProApiKey,
-        logger
+        fixedGasPrice[chainId],
+        "usd"
```

Comment:
I'm pretty sure we can just pass through and use the parent class's constructor here and therefore delete the derived construcotr in PolygonQueries and AlephZeroQueries